docs/conf/prague/2019/news/events-activities.rst

```diff
@@ -0,0 +1,62 @@
+:template: {{year}}/generic.html
+
+.. post:: July 17, 2019
+   :tags: prague-2019, activities, events, tickets, visiting, volunteers
+
+Tickets Update, Events and Activities, Call for Volunteers Reminder
+===================================================================
+
+Greetings documentarians!
+
+The Prague conference is just about two months away, and we are happy to share with you the latest updates.
+
+Tickets are going to sell out soon!
+-----------------------------------
+
+We are now at nearly 75% capacity, and we expect to sell out in the next few weeks, so if you're still on the fence we recommend that you `buy your tickets now <https://www.writethedocs.org/conf/{{shortcode}}/{{year}}/tickets/>`_.
+
+Please make sure you have a conference ticket before you book your travel! We cannot be held responsible for any loss of non-refundable travel expenses, and we cannot risk overselling the venue capacity.
+
+Events and activities
+---------------------
+
+As always, Write the Docs events are more than just talks. Every year we offer a wide range of activities that give our community members different ways to socialize, collaborate, and connect.
+
+Check out what we have in store for you this year in Prague:
+
+- **Boat Tour of Prague, Saturday September 14th**. If you're in town early, you're invited to join us for a 2-hour cruise on the Vltava river, with light snacks and drinks included. This activity is priced separately and isn't included with the conference ticket, so you're welcome to register your friends and family who might be traveling with you. You can find all the details in our `Boat Tour <https://www.writethedocs.org/conf/prague/2019/outing/>`_ page, or register directly on our `ticket website <https://ti.to/writethedocs/write-the-docs-prague-2019>`_.
```

Comment:
Think we're calling it a Boat Ride not Tour. Also are we doing snacks/drinks, I don't remember where we landed on this.

Reply:
We decided to run a tab and pay per consumption on the spot by credit card :)